ui/temboardui/web/routes/settings.py

```diff
@@ -15,3 +15,15 @@ def get_instances():
         role=g.current_user,
         instance_list=orm.Instances.all().with_session(g.db_session).all(),
     )
+
+
+@app.route("/settings/users")
+@admin_required
+def get_users():
+    return render_template(
+        "settings/users.html",
+        nav=True,
+        vitejs=app.vitejs,
+        role=g.current_user,
+        role_list=orm.Roles.all().with_session(g.db_session).all(),
```

Comment:
Can't the second `.all()` be removed? I tried that very quickly and it seems to work.

If so, we should update the other calls to `render_template` in the same file.

Reply:
Le deuxième `.all()` évite de communiquer avec PostgreSQL **dans** le template.